python/GangaLHCb/Lib/Applications/GaudiExec.py

```diff
@@ -269,6 +272,13 @@ def constructExtraFiles(self, job):
 
         folder_dir = master_job.getInputWorkspace(create=True).getPath()
 
+        extraopts = ' '
```

Comment:
Can we at least check if the user has defined extraOpts and if they have warn that this is being added (possibly controlled by a flag as this is much more general purpose than DaVinci). Is there any danger in enabling this by default? could we break existing user scripts?